src/DDTrace/Util/ContainerInfo.php

```diff
@@ -33,7 +33,9 @@ public function __construct($cgroupProcFile = '/proc/self/cgroup')
      */
     public function getContainerId()
     {
-        if (!file_exists($this->cgroupProcFile)) {
+        // We do not want to emit a warning if user application uses ini setting 'open_basedir'
+        // and '/proc/self' is not included in the ini setting.
+        if (!@file_exists($this->cgroupProcFile)) {
```

Comment:
🔝

Reply:
Do we also want to put a warning in dd-doctor specifically for `/proc/self/cgroup` too?